UI/sign_in.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>StackOverflow-lite | Sign In</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="css/main.css" type="text/css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="grid-item header-grid">
+                <header class="header">
+                    <h3>StackOverflow-lite</h3>
+                </header>
+                <nav class="nav-bar">
+                    <li><a href="../index.html">Home</a></li>
+                    <li><a href="dashboard.html">Dashboard</a></li>
+                    <li><a href="profile.html">Profile</a></li>
+                </nav>
+            </div>
+
+            <form action="#" method="" class="grid-item sign-in-form">
+                <div class="form-title sign-in">
+                    <h3>Sign In here</h3>
+                </div>
+                <div class="username">
+                    <label>User name</label>
+                    <input name="username" type="text" required>
+                </div>
+                <div class="password">
+                    <label>Password</label>
+                    <input name="password" type="password" required>
+                </div>
+                <div class="button">
+                    <button name="submit" type="submit">SIGN IN</button>
+                </div>
+
+                <p class="sign-up-title">Have no account, <a href="../index.html" class="sign-up-link">Sign up</a></p>
+            </form>
+
+            <footer class="grid-item footer"><p>Copyright &copy; 2018 SlackOverflow-lite</p></footer>
```

Comment:
Is it a design decision you made for the footer on the sign in and sign up pages not to be at the bottom. If no please make them fixed to the bottom
<img width="1437" alt="screen shot 2018-08-22 at 14 16 47" src="https://user-images.githubusercontent.com/13004199/44460547-30062300-a616-11e8-8c8c-3420bcc7512a.png">

Reply:
No it wasn't a design decision. I actually tried to fix them at the bottom of the page but haven't been able to figure it out. Any suggestions?

Reply:
Yeah. Look into using fixed positioning